### DISCUSSION TITLE
Ultimate version

### DISCUSSION THREAD

**Discussion by qiaoyu113** [qiaoyu113/Aircraft_Cargo_Blance_System#15] [Announcements]
In the final 2.1 version, the code, remarks, readme and other files were optimized on the basis of the 2.0 version; The algorithm rules are optimized.

<hr /><em>This discussion was created from the release <a href='https://github.com/qiaoyu113/Aircraft_Cargo_Blance_System/releases/tag/v2.0.1'>Ultimate version</a>.</em>

**Comment by yuhanliu123** [comment]
fire,fire,fire
